README.md

```diff
@@ -1 +1,95 @@
-# Portfolio-
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portoflio website</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)<br>
+🛠 Built With<br>
+-Tech Stack<br>
+-Key Features<br>
+💻 Getting Started<br>
+-Setup<br>
+-Prerequisites<br>
+- [👥 Authors](#authors)<br>
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portflolio website ] <a name="about-project"></a>
+
+
+*[Portfolio website]* is a website which include my biography in software developement field
+it includes :  <br>
+-my name  <br>
+-Programming languages that I know  <br>
+-My social media perofiles and links  <br>
+🛠 Built With<br>
+**Key Features**<br>
+simple portfolio project, built with html5 and css3 including linter configurattion.<br>
+
+[Set up linters]<br>
+[Add navigation bar for mobile version]<br>
+[Add intro header section for mobile version]<br>
+**Tech Stack**<br>
+This portfolio project, was built with html5 and css3.
+
+Client<br>
+**.HTML**
+**.CSS**
+
+💻 Getting Started<br>
+To get a local copy up and running, follow these steps.<br>
+
+Prerequisites<br>
+In order to run this project you need:<br>
+
+Any IDE or text editor that supports HTML and CSS
+Git installed<br>
+Setup<br>
+Clone this repository to your desired folder:<br>
+
+1. Clone this repository or download the Zip folder:"<br>
+$ git clone <https://github.com/naji-97/Portfolio-/tree/mobile-portfolio)><br>
+2. Navigate to the location of the folder in your machine:<br>
+(https://github.com/naji-97/Portfolio-/tree/mobile-portfolio)<br>
+**3. Press Enter to navigate to your local clone.**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author<a name="authors"></a>
+
+
+👤 *Naji Ali*
+
+- GitHub: [@Naji-](https://github.com/Naji-97)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/naji-ali-55b5421b3//)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+-Create More pages  <br>
+-Adding a screen reader  <br>
+-adding direct emails to my email account  <br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+This is my second project we will  feel loved when you  give us some positive comments and likes 
+
```

Comment:
[OPTIONAL] Please add a link to your license file for this project to make the read me more professional